frontend/packages/topology/stories/91-CollapsibleGroups.stories.tsx

```diff
@@ -145,7 +145,7 @@ const TopologyViewComponent: React.FC<TopologyViewComponentProps> = ({ vis }) =>
             id="collapse-blue"
             label="Collapse Blue"
             isChecked={collapseBlue}
-            onChange={setCollapseBlue}
+            onChange={(checked) => setCollapseBlue(checked)}
```

Comment:
why these changes?

Reply:
Warnings were logged in the console because `onChange` actually has 2 params and the state functions only accept 1.